modules/tile-converter/src/types.ts

```diff
@@ -103,13 +123,25 @@ type AttributeStorageInfo = {
 
 type AttributeValue = {valueType: string; encoding?: string; valuesPerElement?: number};
 
-type Field = {
+export type Field = {
   name: string;
-  type: string;
+  type: ESRIField;
   alias?: string;
   domain?: Domain;
 };
 
+export type ESRIField =
+  | 'esriFieldTypeDate'
+  | 'esriFieldTypeSingle'
+  | 'esriFieldTypeDouble'
+  | 'esriFieldTypeGUID'
+  | 'esriFieldTypeGlobalID'
+  | 'esriFieldTypeInteger'
+  | 'esriFieldTypeOID'
+  | 'esriFieldTypeSmallInteger'
+  | 'esriFieldTypeString'
+  | string;
```

Comment:
If you include string in ESRIField it will match anything. Maybe have a separate `type GeneralField = string`?

Reply:
Removed string. I think it is enough to provide all possible values. Isn't it?